network/utils/network.go

```diff
@@ -0,0 +1,146 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package utils
```

Comment:
It's better to have a better scoped and named package than a generic `utils` package. Can we make this `network/tap` instead? Then, the method can just be `tap.Create`.

Reply:
Yes, exactly my comment above. I'm not sure whether there is value in reusing this logic, but if we do reuse, we should reuse a new coherent "tap" package rather than individual config structs and functions.